AxonDeepSeg/apply_model.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+
+# AxonDeepSeg imports
+import AxonDeepSeg.ads_utils as ads
+from AxonDeepSeg.visualization.merge_masks import merge_masks
+from config import axon_suffix, myelin_suffix, axonmyelin_suffix
+
+from ivadomed import inference as imed_inference
+
+def axon_segmentation(
+                    path_acquisitions_folders, 
+                    acquisitions_filenames,
+                    path_model_folder,
+                    overlap_value=[48,48],
+                    acquired_resolution=None,
+                    verbosity_level = 0
+                    ):
+    '''
+    Segment images using IVADOMED.
+    :param path_acquisitions_folders: the directory containing the images to segment.
+    :param acquisitions_filenames: filenames of the images to segment.
+    :param path_model_folder: path to the folder of the IVADOMED-trained model.
+    :param overlap_value: the number of pixels to be used for overlap when doing prediction. Higher value means less
+    border effects but more time to perform the segmentation.
+    :param acquired_resolution: isotropic pixel size of the acquired images.
+    :param acquired_resolution: isotropic pixel size of the acquired images.
+    :param verbosity_level: Level of verbosity. The higher, the more information is given about the segmentation
+    process.
+    :return: Nothing.
+    '''
+
+    path_model=path_model_folder
+    input_filenames = acquisitions_filenames
+    options = {"pixel_size": [acquired_resolution, acquired_resolution], "overlap_2D":overlap_value, "binarize_maxpooling": True}
```

Comment:
Because of changes in the Microscopy BEP, we will have another required option to pass to `segment_volume`.
It's `pixel_size_units` and must be a string of either `"mm"`, `"um"` or `"nm"`.

It is not implemented yet in ivadomed (draft [PR](https://github.com/ivadomed/ivadomed/pull/1025)) but will be sometime in January.
The current version assumes `"um"` so we could already pass the argument with `"um"`.

Reply:
Resolved in ceadfb